### ISSUE TITLE
[Date Picker] Ensure active user interface components have sufficient contrast (CAL-AMP24) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13532]
**monday.com sync:** #10821961946

### Violation: 
 Ensure active user interface components have sufficient contrast 

The component's year input does not have sufficient contrast or visual indication that the year can be modified.

<img width="358" height="421" alt="Image" src="https://github.com/user-attachments/assets/4a729d52-cfcf-4854-af40-521e9d49c76d" />

### Recommended solution
_TBD, recommend design consultation

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html) 

### Severity: 
 Low 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: CAL-AMP24 Ensure active user interface components have sufficient contrast 
- Module: M3: Form validation_Calcite Feedback Survey 
- Finding ID: [CAL-AMP24](https://esri.hub.essentia11y.com/short-link/3F7ONm8jeZ6dKAgk) 

**Issue**
The border of the user interface component has a contrast ratio below 3.00:1.

Foreground color: #D4D4D4
Background color: #FFFFFF
Contrast ratio: 1.48:1

Examples of this user interface component include:
- 2025 (Year text field in calendar) - on hover

**User Impact**
Users with low vision will have difficulty identifying this content.

**Code Reference** 
 ```
<input inputmode="numeric" maxlength="4" minlength="1" pattern="\d*" type="text" aria-label="Year" class=" year ">

```

**Steps to Reproduce**
1. Locate the user interface component.
2. Inspect it with Chrome DevTools.
3. Review its CSS to determine the foreground and background colors or select the colors of the foreground and background with a color picker.
4. Use a calculator such as https://www.levelaccess.com/color-contrast-checker-new/ to determine the contrast ratio.
5. Notice that the contrast ratio is below 3.00:1.
 
**Recommendation**
Ensure active user interface components have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying components include text field borders, check marks for checkboxes, fillings for radio buttons, focus indicators, and icon-only controls. Non-interactive controls are exempt from this requirement.

To calculate color contrast ratios, use a tool such as the Level Access Accessible Color Picker Chrome extension: https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld or the Color Contrast Checker: https://www.levelaccess.com/color-contrast-checker-new/

**Compliant Code Example**
```
<!-- Example 1: A Send button (with/without a visual boundary) has a focus indicator when focused. The
focus indicator for the component meets the sufficient contrast when the button is focused. -->
---
<!-- Example 2: Visual focus indicator for a link that is a blue (#6699cc) outline around the link. The blue (#6699cc) outline does provide a sufficient contrast with the white (#ffffff) background that is equal to 3 to 1.-->
---
<!-- Example 3: A Submit button at the bottom of a form that is visible, but cannot be activated until all
the required fields in the form are completed does not meet the sufficient color contrast requirement (a disabled control in HTML are not required to meet requirements). -->
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP24.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/3F7ONm8jeZ6dKAgk) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
